bubblecharts/charts/bubbleCloud.js

```diff
@@ -65,104 +65,104 @@ angular.module('myApp')
+          .style('font-size', function (d) { return d.r/scale/2.4})
+          .text(function (d) {return clipText(d, d.name,fontsize/scale/2.5 );})
         }
 
         var zoom = d3.behavior.zoom()
         .scaleExtent([-10, 50])
         .on("zoom", zoomed);
         zoom(svg);
-        
+
         function update(mentionFreq) {
             // console.log("updating");
             // console.log(mentionFreq);
             // if new poll, don't enter, fade out everything
             if(!mentionFreq || Object.keys(mentionFreq).length === 0) {
-                
+
                 chart.selectAll("text")
                 .transition().duration(1000)
                 .attr("opacity", 0)
-                
+
                 chart.selectAll("circle")
                 .transition().duration(1000)
                 .attr("r", 0)
-                
-                window.setTimeout(function(){
+
+                window.setTimeout(function () {
                     // select all bubble groups, remove data then remove dom nodes
                     chart.selectAll('.bubble').data([])
                     .exit().remove();
                 },1001);
-                
+
                 return;
             }
             // if ($scope.mentionFreq) chart.select("#loading").remove();
-            
-            
+
+
             function processData(obj) {
                 var newDataSet = [];
                 for(var prop in obj) {
-                    if(obj[prop] >= $scope.min){
+                    if(obj[prop] >= $scope.min) {
                         var node = {name: prop, className: 'bubbles', size: obj[prop]}
                         newDataSet.push(node);
                     }  
                 }
                 return {children: newDataSet};
-            }     
-            
+            }
+
             // generate data with calculated layout values
             var nodes = bubble.nodes(processData(mentionFreq))
-                .filter(function(d) { return !d.children; }); 
+                .filter(function (d) { return !d.children; }); 
             // filter out the outer bubble for pack layout
             // console.log(nodes);
-            
+
             // input data
             var vis = chart.selectAll('.bubble')
             .data(nodes);
             // console.log(vis);
-            
+
             // add attr to each entered group consisting of <circle> and <text>
             var enter = vis.enter().append('g')
             .attr('class', 'bubble')
-            
+
             enter.append('circle')
-            .attr('r', function(d) { return d.r; })
-            .attr('class', function(d) { return d.className; })
-            .attr('fill', function(d){                 
+            .attr('r', function (d) { return d.r; })
+            .attr('class', function (d) { return d.className; })
+            .attr('fill', function (d) {
                 return color(d.r+10); 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bubblecharts/charts/bubbleCloud.js
+++ b/bubblecharts/charts/bubbleCloud.js
@@ -129,7 +129,7 @@
             .attr('r', function (d) { return d.r; })
             .attr('class', function (d) { return d.className; })
             .attr('fill', function (d) {
-                return color(d.r+10); 
+                return color(d.r+10);
             })
 
 
```